content/en/docs/architecture/release-gating.md

```diff
@@ -34,6 +35,11 @@ Poor candidates include:
 OpenShift CI creates thousands of clusters a week - we can certainly afford to run a few more tests.  However, we want to spread that investment to derive the maximum benefit for everyone, so having OpenShift CI run 30 variations of your exact same tests in slightly different configurations 4 times a day may not be the best way to test.
 {{< /alert >}}
 
+### A new approach to release gating
+Creating useful, stable and testable release gating jobs has proven to be difficult at scale.  The combinatorial explosion of cloud providers, layered products and unique configurations means most jobs have a _>1% flake rate_.  To "meet the tests where they are" the Technical Release Team has created a new type of job called the [_aggregated job_](/docs/release_oversight/improving-ci-signal#job-aggregation-in-a-little-more-detail).
+
+Aggregated Jobs are powerful because they allow a suite of tests to become blocking in spite of a high flake rate.  This is accomplished by analyzing the output from numerous parallel runs to see if payload quality remains as good or better than the historical mean.  If not, the next course of action is to isolate the changes and revert them.  This allows other non-regressing changes to make their way into release.  More importantly it avoids the exponentially difficult task of debugging multiple regressions in parallel (eg, it's hard to debug a layered product regression on a platform that no longer installs properly).
```

Comment:
micro nit: "into release" -> "into the release" or "into releases" or something.

Reply:
updating!